### ISSUE TITLE
Wide-Area Geolocalization with a Limited Field of View Camera. (arXiv:2209.11854v2 [cs.RO] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#26101]
Wide-Area Geolocalization with a Limited Field of View Camera. (arXiv:2209.11854v2 [cs.RO] UPDATED)<br>
https://ift.tt/nt679No<br>
<p>Cross-view geolocalization, a supplement or replacement for GPS, localizes an
agent within a search area by matching images taken from a ground-view camera
to overhead images taken from satellites or aircraft. Although the viewpoint
disparity between ground and overhead images makes cross-view geolocalization
challenging, significant progress has been made assuming that the ground agent
has access to a panoramic camera. For example, our prior work (WAG) introduced
changes in search area discretization, training loss, and particle filter
weighting that enabled city-scale panoramic cross-view geolocalization.
However, panoramic cameras are not widely used in existing robotic platforms
due to their complexity and cost. Non-panoramic cross-view geolocalization is
more applicable for robotics, but is also more challenging. This paper presents
Restricted FOV Wide-Area Geolocalization (ReWAG), a cross-view geolocalization
approach that generalizes WAG for use with standard, non-panoramic ground
cameras by creating pose-aware embeddings and providing a strategy to
incorporate particle pose into the Siamese network. ReWAG is a neural network
and particle filter system that is able to globally localize a mobile agent in
a GPS-denied environment with only odometry and a 90 degree FOV camera,
achieving similar localization accuracy as what WAG achieved with a panoramic
camera and improving localization accuracy by a factor of 100 compared to a
baseline vision transformer (ViT) approach. A video highlight that demonstrates
ReWAG's convergence on a test path of several dozen kilometers is available at
https://youtu.be/U_OBQrt8qCE.
</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
